tools/oesign/CMakeLists.txt

```diff
@@ -22,25 +22,27 @@ if (NOT HAVE_GETOPT_LONG)
   list(APPEND SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/getopt_long.c)
 endif ()
 
-add_executable(oesign ${SOURCES})
+if (NOT COMPONENT MATCHES OEHOSTVERIFY)
```

Comment:
Is this check needed? We already have
```
if (NOT COMPONENT MATCHES OEHOSTVERFIY)
  add_subdirectory(tools)
endif ()
```
in the `CMakeLists.txt`

Reply:
Right, I just tried, but it seems that there are other places where `tools/oesign` can be accessed.